docs/07_tools/02_eosio-helm/tutorial.md

```diff
@@ -0,0 +1,256 @@
+
+```sh
+helm uninstall eosio
+```
+```console
+release "eosio" uninstalled
+```
+
+This might take a few seconds to complete. To check the status we can list the Kubernetes components again:
+
+```sh
+kubectl get pods
+```
+```console
+NAME                   READY    STATUS        RESTARTS   AGE
+eosio-local-nodeos-0   1/1      Terminating   0          55s
+```
+
+Note that for a full cleanup we also need to remove the state generated by the EOSIO node. To that end, first list all Kubernetes persistent artefacts:
+
+```sh
+kubectl get persistentvolumeclaims
+```
+```console
+NAME                       STATUS    VOLUME                                      CAPACITY   ACCESS MODES    STORAGECLASS    AGE
+pvc-eosio-local-nodeos-0   Bound     pvc-dcb06ae4-0bed-41f5-ae55-51beb204c97f    100Gi      RWO             hostpath        71s
+```
+
+Then, delete the actual artefact by specifying the persistent volume claim (pcv) by name `pvc-eosio-local-nodeos-0`:
+```sh
+kubectl delete persistentvolumeclaims pvc-eosio-local-nodeos-0
+```
+```console
+persistentvolumeclaim "pvc-eosio-local-nodeos-0" deleted
+```
+
+[[info | Note]]
+| If you want to retain the EOSIO node state in between launches and cleanups, then do not perform the last step since it will destroy the state stored within the EOSIO persistent volume.
+
+## Part 3: EOSIO Helm Repos
+
+We can create, add, and remove chart repositories locally or to a remote endpoint. A helm repository is where we can collect and share charts remotely, similar to the Linux DPKG, APT, or RPM package managers, but for Kubernetes packages.
+
+Perform any of the following steps to manage our EOSIO Helm repository:
+
+* To list all Helm repositories (local or remote):
+
+```sh
+helm repo list
+```
+```console
+Error: no repositories to show
+```
+
+* To add our `eosio` repository to the list:
+
+```sh
+helm repo add eosio [URL]
+```
+```console
+"eosio" has been added to your repositories
+```
+
+[[info | Note]]
+| Note that you can specify a remote `URL` endpoint as well if you have previously configured it to host a Helm repository. For more information on Helm repositories, refer to the [Helm Chart Repository Guide](https://helm.sh/docs/topics/chart_repository/).
+
+We can check that the `eosio` repository was indeed added to the list:
+
+```sh
+helm repo list
+```
+```console
+NAME    URL
+eosio   localhost:8080
+```
+
+* To pull a specific chart from the repository:
+
+```sh
+helm pull eosio/eosio --untar
+ls -l
+```
+```console
+total 0
+drwxr-xr-x 10 username groupname 320 Apr 07 15:26 eosio
+```
+
+The above command will retrieve the `eosio` chart from the `eosio` repository, unpackage it, and place it under the `eosio` folder (chart name by default).
+
+Now that the chart has been retrieved, we can navigate to the `eosio` chart folder, install it following the steps from [Part 1: Installation](#part-1-installation), and use it following the steps from [Part 2: Basic Usage](#part-2-basic-usage):
```

Comment:
'ing's here and in other places in the content.

Reply:
I think what we disallow is the present continuous tense (where the ing follows a conjugated "to be" verb): I am using (should be: I use). Other cases, such as the gerund use (where the ing'd verb is used as a noun), or the non-present continuous tense (where the "to be" verb is absent, as in this case) are probably fine. However, let's check with V to be sure.

Reply:
as far I understand so far, our rules allow this "the following instructions ..." not a verb.
however "install it following the steps..." is a gerund which we agreed to avoid as much as possible